corehq/apps/domain/decorators.py

```diff
@@ -670,7 +671,10 @@ def _inner(request, *args, **kwargs):
 
 def check_domain_migration(view_func):
```

Comment:
maybe consider renaming this to `check_domain_mobile_access` to make it clearer that it's specifically for mobile endpoints.

Reply:
https://github.com/dimagi/commcare-hq/pull/34587/commits/601b40179e04c62d87907b869cb8c0b552fa2cc7